howso/react_aggregate.amlg

```diff
@@ -462,32 +470,97 @@
 			(assign (assoc conditioned_residuals (true) ))
 		)
 
+		;If we are not always recalculating metrics, then we need to check which metrics exist in cache by attempting to retrieve them
+		(if (and (not always_recalculate) (= conditioned_residuals (false)))
+			(seq
+				(assign
```

Comment:
could you change this to (assign (assoc)) on the same line? then indent output and the code that it evaluates to?